Gaijinizer patch/Map042.txt

```diff
@@ -202,79 +221,87 @@ attack. He also said Hecate ...#MTLed
 『にんげん　いっぱい　なめる　できる　からだに　する』
  言った……。
 > CONTEXT: Map042/events/5/pages/1/50/Dialogue/enemy2
-Before Hecate-sama. "I can make a lot of licks
-because I can do it.". Said…….#MTLed
+Before Hecate-sama... remade me... She said...
+"I'll make you... into something... that can...
+lick lots of... human bodies..."#Ron
 > END STRING
 
 > BEGIN STRING
 わたし　よろこんだ。
 
 カイゾウ　よろこんで　うけた
 > CONTEXT: Map042/events/5/pages/1/55/Dialogue/enemy2
-I am glad. I received a calligraphic call#MTLed
+I was... happy...
+
+Happy to... be remade...#Ron
 > END STRING
 
 > BEGIN STRING
 でも　『シェルター』　きてから　ほとんど……
 
 カマ　つかう　斬る　メイレイ　ばかり
 > CONTEXT: Map042/events/5/pages/1/60/Dialogue/enemy3
-But most of "Shelter"...Kama Use Kill Meira
-Only#MTLed
+But after... arriving at... the shelter... 
+she only... ordered me... to kill with...
+my sickles...#Ron
 > END STRING
 
 > BEGIN STRING
 ……………ウゥ
 
  ユウワク　できる　カラダ　に　もどりたい……
 > CONTEXT: Map042/events/5/pages/1/64/Dialogue/enemy3
-..... w. I would like to return to a body that I
-can do#MTLed
+...Ughh...
+
+I want... my old... seductive body... back...#Ron
 > END STRING
 
 > BEGIN STRING
 （…ひょっとして、あのシェルターにいたヘカーテの部下で
 
 一番可哀想なの、この子なんじゃ……）
 > CONTEXT: Map042/events/5/pages/1/68/Dialogue/kazuya
-(... by chance, in the Hecate's subordinate who
-was in that shelter. It's the most disappointing,
-this girl .........)#MTLed
+(... Don't tell me she was that miserable back in
+the shelter as Hecate's subordinate)#RonIffy
 > END STRING
 
 > BEGIN STRING
 そういえば、確かエンプーサって
 
 『オス』を殺されて怒り狂ったって聞いたけど……
 > CONTEXT: Map042/events/5/pages/1/77/Dialogue/kazuya
-Speaking of that, it's certainly Empuse
-I heard that I was killed and angry at "male" ...#MTLed
+Oh yeah, I remember hearing that there was a
+"male" who was killed and it'd driven you into a
+rage...#Ron
```

Comment:
Tenses are a bit off, we should fix that...

>_**Right, I remember hearing that there was a 
"male" who was killed that had driven you into
a rage...**_